src/app/context/parameters.context.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
```

Comment:
This context is overkill for the current case. Please put this state into the `Home` page and just pass some props to the `Parameters`